R/ci.R

```diff
@@ -20,6 +78,9 @@ NULL
 #' @export
 #' @rdname ci
 use_travis <- function(browse = interactive()) {
+
+  warning("`use_travis()` is deprecated. Please use `use_ci()` in the future.")
```

Comment:
A message feels more appropriate here, I'm not sure we're ready to deprecate `use_travis()` .